manual/manual/refman/exten.etex

```diff
@@ -165,7 +165,7 @@ end = struct
   type t = Leaf of string | Node of ASet.t
   let compare t1 t2 =
     match (t1, t2) with
-    | (Leaf s1, Leaf s2) -> Pervasives.compare s1 s2
```

Comment:
I was working on the manual and trunk broke the build, so I fixed this independently in c578cee935b5b0ecaac8eb3abc1d27fadc79de8c ( plus other minor fixes in dcbf1a8fae9bd313eabd8b9a8ec6511a0ef9ad22 ).

Reply:
Thanks for the cleanup! I rebased this PR and fixed the testsuite in 2ac9f5ff6e5dd2e1 .